### ISSUE TITLE
MLLabelUtils

### ISSUE THREAD

**Issue by tlienart** [dominusmi/warwick-rsg#8]
I'm under the impression that JuliaDB has its own One Hot Encoding facility so that MLlabelutils would effectively become deprecated, is this correct?

**Comment by dominusmi** [comment]
I didn't know about the JuliaDB features so I'll look into that.

On another note, wouldn't those packages still supposedly  be useful, since you wouldn't always want to use JuliaDB? Especially given so many libraries aren't (and probably won't in the near future) be 'compatible' with it.

**Comment by tlienart** [comment]
Of course. However bear in mind the fact that JuliaDB is ambitioned to effectively become the "Pandas of Julia" and therefore all the ML/DS machinery will (eventually) I think be geared towards it in some way or another. 

That being said, many projects do not require huge databases etc and in that case DataFrame.jl or a good old Array would work just fine. I think at this point, it's important for us to understand what would be comptabile or not under the assumption that we'd build the "ML conductor" engine to work with "some data provider" and it would be preferable if we didn't have to accomodate for 5 different data types but rather could assume there was one (maybe 2) that made sense.
